Promact.Oauth.Server/src/Promact.Oauth.Server/Constants/StringConstant.cs

```diff
@@ -7,85 +7,85 @@ namespace Promact.Oauth.Server.Constants
 {
     public class StringConstant
     {
-        public static string ConsumerAppNameDemo = "Demo Name";
-        public static string ConsumerAppNameDemo1 = "Demo Name1";
-        public static string ConsumerAppNameDemo2 = "Demo Name2";
-        public static string ConsumerAppNameDemo3 = "Demo Name3";
-        public static string ConsumerAppNameDemo4 = "Demo Name4";
-        public static string ConsumerAppNameDemo5 = "Demo Name5";
-        public static string ConsumerAppNameDemo6 = "Demo Name6";
-        public static string ConsumerAppNameDemo7 = "Demo Name7";
-        public static string ConsumerAppNameDemo8 = "Demo Name8";
-        public static string TwitterName = "Twitter Name";
-        public static string FaceBookName = "FaceBook Name";
-        public static string ConsumerDescription = "Consumer Description";
-        public static string CallbackUrl = "https://promact.slack.com/messages/@roshni/";
-        public static string CreatedBy = "Ankit";
-        public static string UpdateBy = "Roshni";
-        public static string ATOZ0TO9 = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
-        public static string ATOZaTOz0TO9 = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
-        public static string EmailNotExists = "Email does not exist";
-        public static string ForgotPassword = "Forgot Password";
-        public static string LoginCredentials = "Login Credentials";
-        public static string SuccessfullySendMail = "We have sent you a link on {{emailaddress}} to reset password.Please check your email.";
-        public static string ResetPasswordLink = "${{RestPassWordLink}}$";
-        public static string ResertPasswordUserName = "${{Username}}$";
-        public static string ForgotPasswordTemplateFolderPath = "\\Template\\ForgotPassword.html";
-        public static string UserDetialTemplateFolderPath = "\\Template\\UserDetial.html";
-        public static string From = "From";
-        public static string Password = "Password";
-        public static string Host = "Host";
-        public static string Port = "Port";
-        public static string UserEmail = "${{Email}}$";
-        public static string UserPassword = "${{Password}}$";
-        public static string DefaultUserPassword = "User@123";
+        public const string ConsumerAppNameDemo = "Demo Name";
```

Comment:
I have injected string constant class in controller , repository and test cases. now it should be DI friendly.